panel/src/panel/view.js

```diff
@@ -23,6 +23,14 @@ export default (panel) => {
 	return reactive({
 		...parent,
 
+		async open(feature, options = {}) {
+			if (panel.content.isSaved === false) {
+				await panel.content.save();
```

Comment:
At first sight, I'm not entirely sure what could be a better place. But this seems to me like we need to have at least some sort of toggle to know if the previous view was actually a model view. We probably don't really want this to run on every view change. I.e. you come from the system view and you navigate to the users view.

Reply:
@bastianallgeier What if `content.isSaved` would be always true for non-model views?

Reply:
That's a good idea.